.github/workflows/deploytest.yml

```diff
@@ -52,10 +52,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v4
-    - name: Set up Python 3.9
+    - name: Set up Python 3.10
       uses: actions/setup-python@v5
       with:
-        python-version: 3.9
+        python-version: 3.10
```

Comment:
I should have spotted this, but the actions runs showed it up - this has to be specified as `'3.10'` otherwise it gets parsed as `3.1`

Reply:
oh okay will make the change